guide/onboarding/manual-backup.md

```diff
@@ -0,0 +1,264 @@
+---
+layout: guide
+title: Manual backup
+description: How to backup a recovery phrase manually
+grand_parent: Onboarding
+parent: Backing up a recovery phrase
+nav_order: 2
+permalink: /guide/onboarding/backing-up-a-recovery-phrase/manual-backup/
+main_classes: -no-top-padding
+---
+
+# Manual backup
+
+### How it works
+
+It usually requires users to manually back up their 12 to 24 word recovery phrase by writing it down on a piece of paper and storing it in a safe (but memorable location). In the case that a user’s device breaks or is stolen, the user can recover their funds and wallet by correctly entering their recovery phrase. The [private key management]({{ '/guide/private-key-management/single-user-schemes/#manual-backup--recovery-phrase' | relative_url }}) section dives further into the technical details of this scheme if interested in learning more.
+
+If you are designing a wallet that opts to use manual backups, the following sections outline how to best go about this with new users in mind.
+
+### When to reveal a recovery phrase
+
+The onboarding carousels that we previously mentioned are a great place to prep and explain to users that they will be given a recovery phrase. While some wallets skip this step, in might be appropriate in some scenarios to display it early on. Doing so in the initial onboarding process allows users to pause, internalize what a recovery phrase is, and ensures that they do not neglect this step later on.
+
+{% include /tip/open.html label="Tip: Let users transact even when not backed up" icon="info" color="blue" %}
+
+If you really want to skip showing a user their recovery phrase during the initial onboarding process, create an interaction (i.e. a vibration, modal pop-up, or blocking users from clicking anything) that prompts users to “back up” their recovery phrase at a later time. Perhaps before making any transactions or after several transactions.
```

Comment:
I think I get the general idea, but the phrasing is a little ambiguous. Is it suggesting to *let* users transact when not backed up? Or is it suggesting to *block* users from transacting when not backed up? I feel like the header and paragraph contradict each other.

Reply:
Yeah it's implying you should let users transact regardless, I'll remove the stuff about blocking users. Thanks 👍